### ISSUE TITLE
Add possibility to create not repeatable Long Running Job

### ISSUE THREAD

**Issue by RomaRudyak** [winstongubantes/MatchaBackgroundService#3]
For now there is possibility to add periodic long running jobs.

It would be nice to have an ILongTask or IJobTask which will run only once

**Comment by winstongubantes** [comment]
Hi @RomaRudyak , yes we will gonna include that on our next release. Thanks for the suggestion

**Comment by winstongubantes** [comment]
Hi @RomaRudyak , we have a new update which supports Non-repeatable execution of your IPeriodicTask in version 1.0.5461. We have also updated the documentation. Hope it helps 

https://github.com/winstongubantes/MatchaBackgroundService#create-periodic-task
